pkg/cmd/pipeline/logs.go

```diff
@@ -288,3 +312,17 @@ func lastRun(cs *cli.Clients, ns, pName string) (string, error) {
 	}
 	return latest.ObjectMeta.Name, nil
 }
+
+type byStartTime []v1alpha1.PipelineRun
+
+func (s byStartTime) Len() int      { return len(s) }
+func (s byStartTime) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+func (s byStartTime) Less(i, j int) bool {
+	if s[j].Status.StartTime == nil {
+		return false
+	} else if s[i].Status.StartTime == nil {
```

Comment:
nit: else isn't needed (only a nit)

Reply:
It's needed because it's checking for nil pointer. Otherwise, it panics in certain conditions(similar to a bug which was fixed recently).

Reply:
@khrm 
I meant only the `else` wasn't needed :D 

```go
if foo {
  ___
} else if bar {
  ___
}
```
becomes
```go
if foo {
  ___
} 
if bar {
  ___
}
```
but again it was a `nit` so wasn't a big deal

Reply:
Changed to your suggestion. I also went ahead and changed for `pipelineruns`/`taskruns` as they also used the same sorting approach.